references.go

```diff
@@ -114,6 +108,19 @@ func walkGraph(result *[]*object.Commit, seen *map[plumbing.Hash]struct{}, curre
 	return nil
 }
 
+func derivedFromOneParent(path string, current *object.Commit, parents []*object.Commit) (bool, error) {
```

Comment:
What does `derivedFromOneParent` mean? You are going through all of the parents?

Reply:
Not all parents, maybe I should rename the function to `derivedFromAnyParent`.